test/current_context.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { afterEach, describe, it } from "mocha";
+import * as sinon from "sinon";
+import * as logger from "../src/logger";
+import { expect } from "chai";
+import { createContextEntity } from "../lib/current_context";
```

Comment:
I get error here

Reply:
Please describe error

Reply:
cant find the  path. (pat is not correct)

Reply:
updated

Reply:
not yet :)